base/compiler/abstractinterpretation.jl

```diff
@@ -226,13 +227,13 @@ function abstract_call_gf_by_type(interp::AbstractInterpreter, @nospecialize(f),
                 # the use set for the current SSA value.
                 saved_uses = sv.ssavalue_uses[sv.currpc]
                 sv.ssavalue_uses[sv.currpc] = empty_bitset
-                abstract_call_method(interp, method, csig, match.sparams, multiple_matches, sv)
+                abstract_call_method(interp, method, csig, match.sparams, multiple_matches, si, sv)
```

Comment:
`si` is wrong here

Reply:
`StmtInfo(false)` then and drop the hack to poke the empty bitset into the ssavalue_uses?

Reply:
It looks like there is another one you missed in `abstract_modifyfield!` (the `modifyfield!_tfunc` helper)